# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/crashtests/crashtests.list
# Commit: f2f4628ef591
# Full Hash: f2f4628ef59132f35a4571876952f32f09175847
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-03-26 15:34:27
# Regressor Bug: 1618089
# File Overlap Count: 1
# Description:
#   Bug 1623913 - Make `WSRunObject::PrepareToDeleteRangePriv()` invalidate child of `mScanStartPoint` before reusing it r=m_kato
#   
#   I give up to write a clean patch for this bug with current design.  The trigger
#   is indeed bug 1618089, but this is a hidden regression of bug 1530649.
#   
# ==============================================================================

diff -r 3e5a7430c8d7 -r f2f4628ef591 editor/libeditor/crashtests/crashtests.list
--- a/editor/libeditor/crashtests/crashtests.list	Thu Mar 26 07:00:04 2020 +0000
+++ b/editor/libeditor/crashtests/crashtests.list	Thu Mar 26 05:53:22 2020 +0000
@@ -120,3 +120,4 @@
 load 1574544.html
 load 1596516.html
 load 1618906.html
+load 1623913.html
